lib/logstash/plugin_mixins/rabbitmq_connection.rb

```diff
@@ -34,7 +34,8 @@ def setup_rabbitmq_connection_config
         config :vhost, :validate => :string, :default => "/"
 
         # Enable or disable SSL
-        config :ssl, :validate => :boolean, :default => false
+        # Specify TLS version if using TLS e.g "TLSv1.2"
+        config :ssl, :validate => :string, :default => false
```

Comment:
Actually, a user can either specify "true"/"false" in case he wants to use only ssl for authentication. However, if you specify TLS version in ssl paramter, it expects tls_certificate*

Usage: 
If only SSL , ssl => true
It TLS, ssl => "TLSv1.2"  // Or any other TLS version.
             AND specify tls_cert* paramters also.

This change in code is in accordance with march_hare library code.

-Sunil